crates/nix_rs/src/command.rs

```diff
@@ -40,6 +40,10 @@ pub struct NixCmd {
     #[cfg_attr(feature = "clap", arg(long))]
     pub extra_access_tokens: Vec<String>,
 
+    /// Arguments to pass verbatim to the Nix command
+    #[cfg_attr(feature = "clap", arg(last = true))]
+    pub extra_args: Vec<String>,
```

Comment:
Instead of `extra_args`, just add a `quiet` field looking like this. It is more self-documenting.

https://github.com/clap-rs/clap-verbosity-flag/blob/a7e305e8b0bde21be222ae658cc14813464ea3e6/src/lib.rs#L79-L88